docs/guidance/map/milestones.md

```diff
@@ -0,0 +1,177 @@
+
+The `dueDate`, `dateMet`, `dateModified` and [`status`](../../../schema/codelists/#milestone-status) fields are used to track the lifecycle of the milestone.
+
+## Worked examples
+
+The following worked examples show how to use milestones in different scenarios.
+
+### Planning and tender milestones
+
+The example below includes two OCDS releases:
+
+* A planning release with details of a planned procurement, including the date the budget plan is expected to be ready and the date the tender notice is expected to be issued.
+* A tender release with the actual date the tender notice was issued.
+
+In the planning release:
+
+* The date the budget plan is expected to be ready is represented using a milestone in `planning/milestones` with `.type` is set to 'preProcurement' because the milestone relates to the planning stage of the contracting process. `.dueDate` is set to the date and `.status` is set to 'scheduled'.
+* The date the tender notice is expected to be issued is represented using a milestone in `tender/milestones` because it relates to the tender stage of the contracting process. `.dueDate` is set to the date and `.status` is set to scheduled.
+
+In the tender release:
+
+* The `.dateMet` field in the tender notice milestone is updated with the actual date the notice was issued and `.status` is set to 'met'.
+To explore differences between the planned and actual date of the tender milestone, users can then compare the values of `tender/milestones/dueDate` and `tender/milestones/dateMet` in a single (compiled) release.
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/planning-tender-milestones.json
+   :jsonpointer: 
+   :expand: releases, planning, milestones, tender, milestones
+   :title: planning
+
+.. jsoninclude:: ../../examples/milestones/planning-tender-milestones-2.json
+   :jsonpointer: 
+   :expand: releases, planning, milestones, tender, milestones
+   :title: tender
+
+```
+
+### Contract implementation milestones
+
+The following worked examples show how to use milestones in contracts implementation releases
+
+#### Project data
+
+This worked example shows how to use milestones to model the planned and actual start and finish dates for a construction project.
+
+The example below includes three OCDS releases:
+
+* An award release which includes the scheduled start and end dates for the project.
+* An implementation update release with the actual start date of the project
+* An implementation update release with the actual end date of the project
+
+In the award release:
+
+* The scheduled start and end dates for the project are represented using milestones in `contracts/implementation/milestones` with `.type` set to 'delivery' because they related to the delivery of the contract. The dates are provided in `.dueDate` and `.status` is set to 'scheduled'.
```

Comment:
```suggestion
* The scheduled start and end dates for the project are represented using milestones in `contracts/implementation/milestones` with `.type` set to 'delivery' because they relate to the delivery of the contract. The dates are provided in `.dueDate` and `.status` is set to 'scheduled'.
```